<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaViewerDialog</class>
 <widget class="QDialog" name="MediaViewerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Media viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/play_white.png</normaloff>:/images/play_white.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog  {
	background-color: #111111;
	color: white;
}

QLabel {
	color: white;
}

QPushButton {
	background-color: #25292e;
	color: white;
	border-radius: 5px;
}

QTextEdit, QLineEdit {
	background-color: #363636;
	color: #a8a8a8;
	border: 1px solid #7a7a7a;
}

QComboBox {
	background-color: #25292e;
	color: white;
	border-radius: 5px;
	Padding: 5px;
}

QMessageBox {
	background-color: #111111;
	color: white;
    border: none;
}</string>
  </property>
  <widget class="QWidget" name="mediaviewerwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>680</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>370</width>
     <height>680</height>
    </size>
   </property>
   <widget class="QWidget" name="WDGT_MDVW_VID" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PLBK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/resources.qrc">
      <normaloff>:/images/play_white.png</normaloff>:/images/play_white.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_EXPR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: #a8a8a8;</string>
    </property>
    <property name="text">
     <string>STORY_EXPIRES</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_ACPT">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: #a8a8a8;</string>
    </property>
    <property name="text">
     <string>ACCESSABILITY_CAPTION</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_DWLD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #14c944;
    color: white;
    border: none;
    padding: 6px 12px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #12b03b;
    color: white;
}
QPushButton:pressed {
    background-color: #0f9934;
    color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/resources.qrc">
      <normaloff>:/images/download_white.png</normaloff>:/images/download_white.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_SOUN">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/resources.qrc">
      <normaloff>:/images/volume-on_white.png</normaloff>:/images/volume-on_white.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="WDGT_MDVW_IMG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>500</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
